### PR TITLE
Improve feedback after `upgrade` fail

### PR DESCRIPTION
**Pull request by kasperski95**

### CODE REVIEW

**Comment by MaksymilianDemitraszek** [comment]
You forgot to request review, please remember about it, it makes my life easier 🥺

File: protostar/commands/upgrade/upgrade_command.py [hunk 1]
```diff
@@ -93,12 +94,20 @@ def upgrade(self):
             self._pull_tarball()
             self._install_new_version()
             self.cleanup()
-        # pylint: disable=broad-except
-        except (Exception, KeyboardInterrupt, SystemExit) as err:
+        except KeyboardInterrupt:
+            self._rollback()
+            self.cleanup()
```
Comment (MaksymilianDemitraszek) [inline protostar/commands/upgrade/upgrade_command.py:99]: Why not re-raise?

**Review by MaksymilianDemitraszek — CHANGES_REQUESTED** [review]

File: protostar/commands/upgrade/upgrade_command.py (hunk 1 above)
Comment (kasperski95) [reply protostar/commands/upgrade/upgrade_command.py:99]: There's no point of printing details of this exception. It's an interrupt triggered by a user, not an error.

Comment (MaksymilianDemitraszek) [reply protostar/commands/upgrade/upgrade_command.py:99]: Maybe then it's worth to add a log about interrupted process
`Interrupting...`
cleanup
`Upgrade interupted`

**Pull request merged**